### PR TITLE
React + SIW snippet: replace findDOMNode call with ref

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?** 
  - silence deprecated API warning by using ref
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-210953](https://oktainc.atlassian.net/browse/OKTA-210953)

### CODE REVIEW

**Comment by susanharper-okta** [comment]
ack

File: packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md [hunk 1]
```diff
@@ -68,13 +68,16 @@ Create a `src/OktaSignInWidget.js` file:
 
 ```jsx
 import React, { Component } from 'react';
-import ReactDOM from 'react-dom';
```
Comment (swiftone) [inline packages/@okta/vuepress-site/code/react/okta_react_sign-in_widget/index.md:71]: We should also update this page to say "this example uses okta-react vX.Y.Z, react 16.7.x, and react-router 5.x" (with whatever the actual version numbers are)

**Review by swiftone — APPROVED** [review]

**Comment by jakubvul-okta** [comment]
This is not related to the site redesign, it's an update to our React docs, so I've removed the label.

**Comment by oleksandrpravosudko-okta** [comment]
superseded by https://github.com/okta/okta-developer-docs/pull/1739

**Pull request closed without merging**